packages/datastore-storage-adapter/__tests__/ExpoSQLiteDatabase.test.ts

```diff
@@ -0,0 +1,535 @@
+import ExpoSQLiteDatabase from '../src/ExpoSQLiteAdapter/ExpoSQLiteDatabase';
+import { ParameterizedStatement } from '../src/common/types';
+
+jest.mock('expo-sqlite', () => ({
+	openDatabase: jest.fn(() => mockDatabase),
+}));
+
+jest.mock('expo-file-system', () => ({
+	deleteAsync: jest.fn(path => null),
+	documentDirectory: '/sample/path',
+}));
+
+const mockDatabase = {
+	readTransaction: jest.fn(),
+	transaction: jest.fn(),
+	_db: { close: () => {} },
```

Comment:
Think this should be a `jest.fn()` and there ought to be assertions on it being called where appropriate (e.g. in the `clear` test)